apache/incubator-zeppelin
# [Zeppelin 537] Improve behavior of reloading notebooks from storage

### What is this PR for?
Improve behavior of reloading notebooks from storage

### What type of PR is it?
Improvement

### Is there a relevant Jira issue?
[ZEPPELIN-537](https://issues.apache.org/jira/browse/ZEPPELIN-537?jql=project%20%3D%20ZEPPELIN)

### How should this be tested?
1. Add new `notebookId/note.json` under `ZEPPELIN_HOME/notebook`
2. Click reload button next to 'Notebook'
3. Check if the new added note is loaded

### Screenshots (if appropriate)
![reload fig](https://cloud.githubusercontent.com/assets/8503346/12011889/283c8e9c-ac96-11e5-81f6-4cdb305020d3.gif)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
